docs/install/install_everest_helm_charts.md

```diff
@@ -89,23 +101,54 @@ Here are the steps to install Percona Everest and deploy additional database nam
                 everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:8080/TCP    10s
                 ```
 
+        3. Run the following command to connect to Percona Everest when **TLS is enabled**:       
+                    
+            ```sh
+            kubectl patch svc/everest -n everest-system -p '{"spec": {"type": "LoadBalancer"}}'
+            ```
+
+            ??? example "Expected output"
+                ```sh 
+                kubectl get svc/everest -n everest-system
+                ```
+                ```
+                NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+                everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:443/TCP    10s
+                ```
+
+            This is the address where you can then launch Everest at the end of the installation procedure. In this example, the external IP address used is [http://34.175.201.246](http://34.175.201.246).
+
+            For comprehensive instructions on enabling TLS for Percona Everest, see the section [TLS setup with Percona Everest](../security/tls_setup.md#tls-setup-with-percona-everest).
 
     === "Node Port"
         A NodePort is a service that makes a specific port accessible on all nodes within the cluster. It enables external traffic to reach services running within the Kubernetes cluster by assigning a static port to each node's IP address.
+        {.power-number}
 
         1. Run the following command to change the Everest service type to `NodePort`:
 
             ```sh
             kubectl patch svc/everest -n everest-system -p '{"spec": {"type": "NodePort"}}
             ```
-        2. The following command displays the port assigned by Kubernetes to the everest service, which is `32349` in this case.
+            The following output displays the port assigned by Kubernetes to the everest service, which is `32349` in this case.
 
             ```sh
             kubectl get svc/everest -n everest-system
             NAME      TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
             everest   NodePort   10.43.139.191   <none>        8080:32349/TCP   28m
             ```
 
+        2. Run the following command to change the Everest service type to `NodePort` when **TLS is enabled**
+
+            The following command displays the port assigned by Kubernetes to the everest service, which is `443` in this case.
+
+                `
+                kubectl get svc/everest -n everest-system
+                NAME      TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+                everest   NodePort   10.43.139.191   <none>        8080:443/TCP   28m
+            
+
+            For comprehensive instructions on enabling TLS for Percona Everest, see the section [TLS setup with Percona Everest](../security/tls_setup.md#tls-setup-with-percona-everest).
```

Comment:
Same here, the command does not change, so I don't see why we have a separate step. Only the ports change in the output, so this is confusing. 

Please also double check the ports, it should be `443:8080` if I am not wrong

Reply:
Will check it.